.travis.yml

```diff
@@ -91,10 +100,11 @@ script:
   - sbt ++$TRAVIS_SCALA_VERSION compile
   - |
     if [[ "$REPOSITORY_NAME" = "hubat" ]]; then
+      # TODO: Update test next
       sbt ++$TRAVIS_SCALA_VERSION test:compile
       sbt ++$TRAVIS_SCALA_VERSION "project hat" coverage test -Dconfig.file=hat/conf/application.test.conf
     fi
-  - sbt -Denv=prod docker:stage
+  - sbt -Denv=prod -Dhttp.port=8080 -Dpidfile.path=/dev/null -Dplay.server.pidfile.path=/dev/null docker:stage
```

Comment:
what is the intent here ?
These java variables get passed to the docker:stage task, but my understanding is that they WILL NOT be set when you run the container. An option which will make them consistently available is adding them to the build.sbt i.e.  javaOptions in Universal ++= Seq("-Dhttp.port=8080")

Reply:
This is the legacy of different attempts to debug build issues. Suggest leaving it there for the current merge as we have confirmed working build process. Once merged we should clean up on dev and propagate to other branches.